core/templates/dev/head/pages/exploration_editor/editor_tab/StateResponses.js

```diff
@@ -595,6 +597,9 @@ oppia.controller('StateResponses', [
             $scope.trainingDataFeedback = '';
             $scope.trainingDataOutcomeDest = '';
 
+            // Inject textInputRulesService dynamically
+            textInputRulesService = $injector.get('textInputRulesService');
```

Comment:
Haha, we could do just that. One thing we need to make sure is that we always name the rule service as per the interaction ID. 

Thanks for the suggestion!